src/main/php/webservices/rest/RestResponse.class.php

```diff
@@ -58,6 +58,30 @@ public function location() {
     return null;
   }
 
+  /**
+   * Returns cookies sent by server
+   *
+   * @see    https://tools.ietf.org/html/rfc6265
+   * @return [:webservices.rest.Cookie]
+   */
+  public function cookies() {
+    if (!isset($this->lookup['set-cookie'])) return [];
+
+    $cookies= [];
+    foreach ($this->headers[$this->lookup['set-cookie']] as $cookie) {
+      $attributes= [];
+      preg_match('/([^=]+)=("([^"]+)"|([^;]+))?(;(.+))*/', $cookie, $matches);
+      if (isset($matches[6])) {
+        foreach (explode(';', $matches[6]) as $attribute) {
+          $r= sscanf(trim($attribute), "%[^=]=%[^\r]", $name, $value);
+          $attributes[$name]= 2 === $r ? urldecode($value) : true;
+        }
+      }
+      $cookies[$matches[1]]= new Cookie($matches[1], isset($matches[2]) ? urldecode($matches[2]) : null, $attributes);
```

Comment:
There is no handling of __Secure and __Host prefixes here, see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie

Reply:
✅  Done